cpr/session.cpp

```diff
@@ -291,6 +293,25 @@ void Session::Impl::SetVerifySsl(const VerifySsl& verify) {
     }
 }
 
+void Session::Impl::SetProtocolVersion(const ProtocolVersion& protocolversion) {
+#if LIBCURL_VERSION_MAJOR >= 7
```

Comment:
noop'ing (or ignoring) this call silently based on the libcurl version is maybe a little surprising/unintuitive; thoughts on changing this to throw an exception if the protocol version isn't supported?

Reply:
This is a tricky subject, but I think the right thing is to throw an exception here if the curl version doesn't support HTTP/2. Having clients send HTTP/1.x requests when they think they're sending HTTP/2 would be pretty bad